cypress/e2e/CMS/WYSIWYG/$EmbedFeaturedContent.feature

```diff
@@ -0,0 +1,201 @@
+            | Feature Card Description | Automated Test Article for Featured Content Test - Feature Card Desc | field_feature_card_description |
+        And user enters "Article Heading1" as 1 body section heading
+        And user fills out 1 "Body" text area with "This article will be used for testing Featured Content"
+        And user selects 1 Lead Image from the list of images
+        And browser waits
+        And user selects 3 Promotional Image from the list of images for featured content article
+        And browser waits
+        And user remembers the source of selected promo image for further verification in the featured content article
+        And user selects "Published" from "Save as" dropdown
+        Then user saves the content page
+
+
+    Scenario: User is adding new test Blog post content type for embedding it as a Featured Content
+        Given user is navigating to "/user/login"
+        When user enters credentials
+        And user clicks "Log in" button
+        Then user is logged in and the user name "admin" is displayed in the toolbar
+        And the tool bar appears at the top
+        When user clicks on "Content" tab
+        And user clicks on "Add content" action button
+        And user clicks on "Blog Post" content type
+        Then page title is "Create Blog Post"
+        And user selects "Cancer Currents" from Blog Series dropdown
+        And browser waits
+        And user fills out the following fields
+            | fieldLabel               | value                                                                  | field_name                     |
+            | Pretty URL               | blog-post-for-testing                                                  | field_pretty_url               |
+            | Page Title               | Automated Test Blog Post for Featured Content Test                     | title                          |
+            | Browser Title            | Automated Test Blog Post for Featured Content Test - Browser Title     | field_browser_title            |
+            | Card Title               | Automated Test Blog Post for Featured Content Test - Card Title        | field_card_title               |
+            | Meta Description         | Automated Test Blog Post for Featured Content Test - Meta Description  | field_page_description         |
+            | Feature Card Description | Automated Test Blog Post for Featured Content Test - Feature Card Desc | field_feature_card_description |
+            | Author                   | Automated Test Blog Post for Featured Content Test Author              | field_author                   |
+        And user fills out "Body" text area with "This blog post will be used for testing featured content"
+        And user selects 1 Lead Image from the list of images
+        And browser waits
+        And user selects 3 Promotional Image from the list of images for blog post
```

Comment:
also remove promo image from blog - lead image should display instead on FF if you dont provide a promo override
article will have a promo override and blog should display lead ( again 2 use cases)

Reply:
done